ftw/contentnav/behaviors/content_categories.py

```diff
@@ -66,6 +66,13 @@ def new_content_categories(self, value):
             return ()
         else:
             categories = tuple(self.annotation_storage.content_categories)
+
+            # Strip all new categories and remove duplicates.
```

Comment:
This does not yet remove duplicates, this is done below.
Why did you invent a new variable rather than stripping inplace? Just curious 😉